testplan/testing/base.py

```diff
@@ -488,11 +490,19 @@ class ProcessRunnerTest(Test):
     _VERIFICATION_TESTCASE_NAME = "ExitCodeCheck"
     _MAX_RETAINED_LOG_SIZE = 4096
 
-    def __init__(self, **options):
+    def __init__(self, pre_args=None, post_args=None, **options):
         proc_env = os.environ.copy()
         if options.get("proc_env"):
             proc_env.update(options["proc_env"])
         options["proc_env"] = proc_env
+        if pre_args:
+            options["pre_args"] = pre_args
+        else:
+            options["pre_args"] = []
+        if post_args:
+            options["post_args"] = post_args
+        else:
+            options["post_args"] = []
```

Comment:
If we remove pre_args & post_args from the parameter list, then they gets to **options, and will be updated to self.cfg.pre_args/post_args automatically when Entity class's __init__ is  called. Then we don't need these few lines.

Reply:
Removed.